client/app/bundles/course/assessment/submission/pages/SubmissionsIndex/index.jsx

```diff
@@ -61,7 +61,11 @@ class VisibleSubmissionsIndex extends React.Component {
     this.state = {
       publishConfirmation: false,
       includePhantoms: false,
-      tab: 'my-students-tab',
+      tab: props.submissions.some(
```

Comment:
Last time I tried writing the initialisation logic this way but it seems that when the submission prop is fully loaded, react doesnt update the state.tab. As a result, it always defaults to students-tab for the histogram even tho my-students exist.. 

See below: The initial histogram is not correctly generated and thats why the switch case below was written the way it was

![image](https://user-images.githubusercontent.com/42570513/136756753-3bfd7091-fed3-4a65-bfea-872a652337b0.png)